contrib/epee/include/net/abstract_tcp_server.h

```diff
@@ -38,6 +38,9 @@
 
 #pragma comment(lib, "Ws2_32.lib")
 
+#undef MONERO_DEFAULT_LOG_CATEGORY
+#define MONERO_DEFAULT_LOG_CATEGORY "net"
```

Comment:
Is this a good idea in a header file? If someone includes this file, but forgets to set their own category ...